src/commands/releases.rs

```diff
@@ -516,6 +516,24 @@ fn execute_set_commits<'a>(
     Ok(())
 }
 
+fn execute_set_manual_commits<'a>(ctx: &ReleaseContext<'_>, matches: &ArgMatches<'a>) -> Result<(), Error> {
+    let version = matches.value_of("version").unwrap();
+    let org = ctx.get_org()?;
+
+    if let Ok(projects) = ctx.get_projects(matches) {
```

Comment:
Not sure if this error should be handled differently. If we don't know if this matters I would even opt for `unwrap()` or `?`

Reply:
```
let projects = ctx.get_projects(matches)?;
```

Reply:
This was apparently copied from another fn which may have to be fixed too